requirements/frozen.txt

```diff
@@ -337,9 +338,9 @@ webencodings==0.5.1
 websocket-client==0.40.0
 Werkzeug==0.16.0
 widgetastic.core==0.51
-widgetastic.patternfly==1.2.2
+widgetastic.patternfly==1.3.0
```

Comment:
Please update frozen_docs as well.  How did `magnitude` make it in?

Reply:
My local .cfme_venv directory wasn't clean. Fixed frozen.txt. As discussed in chat, miq requirement doesn't touch frozen_docs.py3.txt, so we'll leave that file alone for now.